eccehomo/app.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
@maikthomas @fmartingr `import os.path` :question: 
http://stackoverflow.com/questions/2724348/should-i-use-import-os-path-or-import-os